pandas-stubs/_typing.pyi

```diff
@@ -427,12 +427,28 @@ ByT = TypeVar(
     Interval[Timedelta],
     tuple,
 )
+SeriesByT = TypeVar(
```

Comment:
Can you please add a comment here that `SeriesByT` is the intersection of `S1` and `ByT`. If either of them is adjusted in the future, it is then also clear that `SeriesByT` needs to be adjusted.